connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -73,23 +77,33 @@
                     " header values to strings and deserialize them by inferring the schemas.";
     public static final String HEADER_CONVERTER_CLASS_DEFAULT = SimpleHeaderConverter.class.getName();
 
+    /**
+     * @deprecated As of 1.2.0
```

Comment:
There's some confusion with the versioning, but currently this is planned to be 2.0.0 due to JDK bump.